src/components/molecules/TemplateFormRenderer/TemplateFormRenderer.tsx

```diff
@@ -93,6 +111,21 @@ const TemplateFormRenderer: React.FC<IProps> = props => {
 
   return (
     <TemplateFormErrorBoundary>
+      {displayLink && (
+        <S.ErrorContainer>
+          <S.ErrorText>
+            <S.ReadMoreLink
+              onClick={() => {
+                window.open('https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names');
```

Comment:
Lets use `shell.openExternal` instead of `windowsopen`

Reply:
Done @topliceanurazvan !